### PR TITLE
修复遍历不到jar的问题

### PR DESCRIPTION
**Pull request by 2535688890**

### CODE REVIEW

**Comment by dlenski** [comment]
:confused: 
What problem does this solve?

File: zxing/__init__.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def __init__(self, classpath=None, java=None):
         else:
             self.classpath = os.path.join(os.path.dirname(__file__), 'java', '*')
 
-        for fn in chain.from_iterable(glob.glob(cp) for cp in self.classpath.split(':')):
+        for fn in glob.glob(self.classpath):
```
Comment (dlenski) [inline zxing/__init__.py:55]: Your code change here would prevent classpath lists containing multiple `:`-delimited paths from work.

It literally does _nothing besides that_

Comment (2535688890) [reply zxing/__init__.py:55]: On Windows, use python 3.8.9 python 3.12.0 cannot get the core.jar in the Java/* path

Comment (2535688890) [reply zxing/__init__.py:55]: classpath.split(':')  Will separate "D: / python / xxx / zxing" into "D" and "/ python / xxx / zxing"

Comment (dlenski) [reply zxing/__init__.py:55]: Got it.

How about requiring the classpath separator to be `;` on Windows, and `:` on other OSes, as is apparently Java's own standard? https://stackoverflow.com/a/60211688

**Pull request closed without merging**